testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2997,6 +2997,19 @@ public void testInsertArray()
         }
     }
 
+    @Test
+    public void testInsertSameValues()
```

Comment:
Looks useful but I don't understand what component this test is trying to exercise - at-least it's not obvious to me.

Reply:
Thanks for explaining.